pkg/flow/internal/stdlib/stdlib.go

```diff
@@ -1,46 +0,0 @@
-// Package stdlib contains Flow-specific standard library functions exposed to
-// River configs.
-package stdlib
-
-import (
-	"encoding/json"
-
-	"github.com/grafana/agent/component/discovery"
-	"github.com/prometheus/prometheus/discovery/targetgroup"
-)
-
-// Identifiers holds a list of stdlib identifiers by name. All interface{}
-// values are River-compatible values.
-//
-// Function identifiers are Go functions with exactly one non-error return
-// value, with an optionally supported error return value as the second return
-// value.
-var Identifiers = map[string]interface{}{
```

Comment:
Should we keep this file around for future usages. I know https://github.com/grafana/agent/pull/4388/files uses it for one thing, unless this is the wrong place to add those. I still think its possible there are flow-specific stdlib functions possible.

Reply:
#4388 is extending pkg/river/internal/stdlib, but this is removing pkg/flow/internal/stdlib. We consider it an anti-pattern now to have Flow-specific stdlib functions, which is why this file is being removed in favor of only having the River one.